ios/RNSScreenStackHeaderSubview.mm

```diff
@@ -22,6 +22,15 @@ @implementation RNSScreenStackHeaderSubview
 
 #pragma mark - Common
 
+// We're forcing the navigation controller's view to re-layout
+// see: https://github.com/software-mansion/react-native-screens/pull/2385
+- (void)ensureNavigationLayout
```

Comment:
`ensureNavigationBarLayout`? or `layoutNavigationBarIfNeeded`? - but only if you limit layouting to the navigation bar and not whole navigation controller content view. Otherwise I suggest `ensureNavigationControllerContentLayout` or `layoutNavigationControllerIfNeeded`.

Reply:
https://github.com/software-mansion/react-native-screens/pull/2385/commits/9179b6b6aa8afe9626c453d4d99d27b84bbd3595